integration-tests/deploy_tensorflow_models.py

```diff
@@ -0,0 +1,200 @@
+from __future__ import absolute_import, print_function
+import os
+import sys
+import requests
+import json
+import numpy as np
+import time
+import logging
+
+cur_dir = os.path.dirname(os.path.abspath(__file__))
+
+sys.path.insert(0, os.path.abspath('%s/util_direct_import/' % cur_dir))
+from util_package import mock_module_in_package as mmip
+import mock_module as mm
+
+import tensorflow as tf
+
+from test_utils import (create_docker_connection, BenchmarkException, headers,
+                        log_clipper_state)
+cur_dir = os.path.dirname(os.path.abspath(__file__))
+sys.path.insert(0, os.path.abspath("%s/../clipper_admin" % cur_dir))
+from clipper_admin.deployers.tensorflow import deploy_tensorflow_model, create_endpoint
+
+logging.basicConfig(
+    format='%(asctime)s %(levelname)-8s [%(filename)s:%(lineno)d] %(message)s',
+    datefmt='%y-%m-%d:%H:%M:%S',
+    level=logging.INFO)
+
+logger = logging.getLogger(__name__)
+
+app_name = "tensorflow-test"
+model_name = "tensorflow-model"
+
+
+def normalize(x):
+    return x.astype(np.double) / 255.0
+
+
+def objective(y, pos_label):
+    # prediction objective
+    if y == pos_label:
+        return 1
+    else:
+        return 0
+
+
+def parseData(train_path, pos_label):
+    trainData = np.genfromtxt(train_path, delimiter=',', dtype=int)
+    records = trainData[:, 1:]
+    labels = trainData[:, :1]
+    transformedlabels = [objective(ele, pos_label) for ele in labels]
+    return (records, transformedlabels)
+
+
+def reset_vars():
+    sess.run(tf.global_variables_initializer())
+
+
+def reset_tf():
+    global sess
```

Comment:
Can you make `sess` a local variable that gets passed around instead of a global?

Reply:
Done